apps/src/templates/sectionProgress/progressTables/ProgressTableContainer.jsx

```diff
@@ -191,7 +193,7 @@ class ProgressTableContainer extends React.Component {
     this.setState({rows});
   }
 
-  onRow = row => {
+  onRow(row) {
```

Comment:
JS newbie question -- should we only bind `this` when we need it (i.e. when the function references it) or should we always bind (to reduce confusion)?

Reply:
only bind when we need `this` (or use an arrow function, i've abandoned my proposal that we standardize a pattern for this)